### DISCUSSION TITLE
Dec 2023 Review - PR5: Graphs of Polynomial Functions

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#47] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html

Author: @tdegeorge

Primary Reviewer: @AbbyANoble (feedback by Dec 11 morning)

**Comment by StevenClontz** [comment]
(@siwelwerd thanks for doing this - slipped away from me last week...)

**Comment by AbbyANoble** [comment]
Great overall! Here are some things I noticed:

In [Activity 1.5.1](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#activity-Identify-PF-from-graph), I’m concerned that if we asked about an exponential or log function they may classify it as a polynomial based on the smooth and continuous definition. Maybe some more graphs included, like rational, exponential, log, so that doesn’t happen. Make sure the asymptotes are obvious to lead to the discussion that polynomials won’t have those. Then adjust [Remark 1.5.2](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#characteristics-of-polynomial-graphs) to fit.

[Activity 1.5.3](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#activity-characteristics-from-polynomial-graph) - I like how (f) and (g) have multiple answers to get students to use the different ways to communicate the same idea!

[Activity 1.5.5](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#use-information-to-sketch-graph-of-polynomial) The numbers in part (a) and (b) aren’t in math mode (<m> </m>)


A couple things that could be added:

- Activity where they have the function written out and have to graph it (assuming no graphing calculator) to see the connection between the function and the graph. Maybe repeat the questions from [Activity 1.5.3](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#activity-characteristics-from-polynomial-graph) given f(x)=[some other polynomial, degree 4 or 5 maybe], and then ask them to sketch it. Would be a good gallery walk! And maybe skip the min/max questions? (I'm not sure how we are getting them to estimate those?)

- Exploring how the multiplicity of a zero connects to whether it crosses or “bumps” the axis. Being able to include that in their graph accurately.


Also, and this isn’t part of your writing, just while I’m thinking about it I’ll throw it here. I’m unsure on the part of the objective about estimating locations of maxima and minima. How are we expecting them to do that?

**Comment by siwelwerd** [comment]
My big question is about alignment with the learning objective.  These activities give students a graph, and then they determine information about it.  However, the objective says 

> Find the intercepts, estimated locations of maxima and minima, and end behavior of a polynomial function, and *use this information to sketch the graph*.

This suggests to me something much more like the [Checkit exercises](https://teambasedinquirylearning.github.io/precalculus/2024/exercises/#/bank/PR5).  So we probably need to either rethink the learning objective, or add in activities more aligned with the objective after these initial explorations.

I also think either way this section would benefit from some additional fluency builders at the end.

**Comment by tdegeorge** [comment]
@siwelwerd @AbbyANoble I looked at your comments and made some changes....There are still pieces I need to add in, like the max/min stuff, which I think we need to talk about as a group. I added in a few more activities that align with the objective but left the activities I had originally as well because I am not sure where that goes...

**Comment by siwelwerd** [comment]
I think this is really great through [Activity 1.5.13](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#PR5-activity-5).  However instead of the remainder, I think you'll want to jump into some fluency building there.  Give them another polynomial and ask the same questions (maybe twice).

I do like [Activity 1.5.18](https://teambasedinquirylearning.github.io/precalculus/2024/PR5.html#use-information-to-sketch-graph-of-polynomial) too.  But right now I think just take out 1.5.14--1.5.17.  What do you think @AbbyANoble?

**Reply by tdegeorge** [reply]
Between definition 1.5.6 and definition 1.5.7, I left a note about having another multiplicity activity and then wrote a note about max/min. Any ideas on what I should do there? I also wasn't sure about how much we wanted to include the max/min. Like how do others have their students find it? The objective says they should be able to find "estimated locations" of it....I don't normally ask students to do that in my class... @siwelwerd @AbbyANoble

**Reply by siwelwerd** [reply]
I think a simple fluency building activity around multiplicity is all that's needed.

For max/min, I think maybe just being able to identify an extrema at a zero (because it has even multiplicity), or identifying an extrema between two zeroes; this is in line with the [current Checkit problem](https://teambasedinquirylearning.github.io/precalculus/2024/exercises/#/bank/PR5/1/) @TienChih wrote

**Reply by AbbyANoble** [reply]
I actually really like 1.5.14, especially with the added part about exponential functions. I may would change it a little where it doesn't assume they know what an exponential function is already. You could also appeal to the end behavior to justify it not being a polynomial since at this point they haven't really dealt with asymptotes.

**Reply by AbbyANoble** [reply]
I'm also confused about including the max/min. In college algebra/precalculus, I only have them determine a max/min by estimating from a picture, not from the function written out. And I would think that [Section 1.4: Zeroes of Polynomial Functions (PR4)](https://teambasedinquirylearning.github.io/precalculus/2024/PR4.html#PR4) 
would need an activity about multiplicity as related to the function's notation. (I'm seeing mostly fluency builders there, not scaffolded exploration.)

**Reply by AbbyANoble** [reply]
Then you could do scaffolded exploration multiplicity activity where you write the function and show the graph at the same time to get students to connect odd multiplicity with going through the graph, but even multiplicity with bumping the graph.

**Reply by tdegeorge** [reply]
Added in a multiplicity activity and a max/min activity/remark....I left the other activities because it looks like we may need to talk about some of these as a group?
